action.py

```diff
@@ -23,11 +23,11 @@
 from calibre.devices.usbms.driver import debug_print
 
 try:
-    from PyQt5.Qt import (pyqtSignal, Qt, QApplication, QIcon, QMenu, QPixmap,
+    from PyQt5.Qt import (pyqtSignal, Qt, QApplication, QIcon, QMenu, QMessageBox, QPixmap,
```

Comment:
Please remove code, that is now extracted into separate PR's like #25 and #24.

Reply:
This will land last and I'll rebase, I don't need to remove it here. I'd wait to review this PR until those others merge.

Reply:
OK.